src/components/Watch.svelte

```diff
@@ -127,29 +132,29 @@
       <div
         class="tile autoscale"
         bind:this={chatSideElem}
-        style={$videoSide == VideoSide.TOP ? 'min-width: 100% !important;' : ''}
+        style={isTopSide ? 'min-width: 100% !important;' : ''}
       >
         <div
-          class="flex {$chatSplit == ChatSplit.VERTICAL
+          class="flex {isChatVertical
             ? 'vertical'
             : 'horizontal'}"
           style="
-            {$videoSide == VideoSide.RIGHT ? 'width: calc(100% - 10px);' : ''}"
+            {isRightSide ? 'width: calc(100% - 10px);' : ''}"
         >
           <div
             class="tile resizable"
-            style="{$chatSplit == ChatSplit.VERTICAL
+            style="{isChatVertical
               ? 'width'
               : 'height'}: {$chatSize}%;
-                min-{$chatSplit == ChatSplit.VERTICAL
+                min-{isChatVertical
               ? 'width'
               : 'height'}: 10px;
```

Comment:
Just use one ``isChatVertical`` ternary expression to set both ``width|height`` and ``min-width|min-height``.